odoo/tools/config.py

```diff
@@ -549,7 +550,7 @@ def die(cond, msg):
             self.save()
 
         # normalize path options
-        for key in ['data_dir', 'logfile', 'pidfile', 'test_file', 'screencasts', 'screenshots', 'pg_path', 'translate_out', 'translate_in', 'geoip_city_db', 'geoip_country_db']:
+        for key in ['data_dir', 'logfile', 'logfilesdb', 'pidfile', 'test_file', 'screencasts', 'screenshots', 'pg_path', 'translate_out', 'translate_in', 'geoip_city_db', 'geoip_country_db']:
             self.options[key] = self._normalize(self.options[key])
```

Comment:
Have you verified that it works on windows? IIRC `%` is treated as an escape sequence (e.g. `%USERNAME%`)

Reply:
Works as intended with both config key and as CLI parameter.

However, it create a new issue with "?" as Windows does not allow this character into a file name:
`OSError: [Errno 22] Invalid argument: 'c:\\program files\\odoo 17.0.20240506\\server\\db-?.log'`